testing/matrix/linter/rules/rules.go

```diff
@@ -166,8 +172,22 @@ func containerEnvVariablesDuplicates(object storage.StoreObject, containers []v1
 	return errors.EmptyRuleError
 }
 
-func containerImageTagLatest(object storage.StoreObject, containers []v1.Container) errors.LintRuleError {
+func shouldSkipModuleContainer(module string, container string) bool {
+	if module == "okmeter" && container == "okagent" {
+		return true
+	}
+	if module == "d8-control-plane-manager" && strings.HasPrefix(container, "image-holder") {
+		return true
+	}
+	return false
+}
+
+func containerImageTagCheck(object storage.StoreObject, containers []v1.Container) errors.LintRuleError {
 	for _, c := range containers {
+		if shouldSkipModuleContainer(object.Unstructured.GetName(), c.Name) {
+			continue
+		}
+
 		imageParts := strings.Split(c.Image, ":")
```

Comment:
I'd like to avoid splitting and suggest using something like this https://github.com/google/go-containerregistry/blob/45aaa6c000ebf2acacad9beef7c82667ec51662c/pkg/name/ref.go#L39-L49

Reply:
i used go-containerregistry functions to parse image string, thanks !